src/test/scala/strawman/collection/test/TraverseTest.scala

```diff
@@ -69,9 +69,8 @@ class TraverseTest {
 
   @Test
   def eitherSequenceTest: Unit = {
-    val xs3 = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
+    val xs3: mutable.ListBuffer[Either[Int, String]] = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
```

Comment:
The problem is the type To in ListBuffer, which is non-variant. So even if we could widen, this would not work in this case.